### ISSUE TITLE
Add 32-bit bundled linux namecoind

### ISSUE THREAD

**Issue by yetanotherusernamebecausegithubisbugged** [namecoin/Convergence#36]
Please add 32-bit bundled linux namecoind. I had spent a hour to figure out why bundled namecoind weren't working and to compile it by myself. Not a good experience for a common end-user.

**Comment by JeremyRand** [comment]
Yep, agreed.  A lot of this code is being refactored at the moment, but any future bundles should support 32-bit Linux.  Sorry it frustrated you, I'll see if we can document this better for the meantime.
